erpnext/accounts/page/pos/pos.js

```diff
@@ -616,13 +616,23 @@ erpnext.pos.PointOfSale = erpnext.taxes_and_totals.extend({
 			? this.item_serial_no[this.child.item_code][0] : '');
 	},
 
+	update_paid_amount_status: function(update_paid_amount){
+		update_paid_amount_dict = {true: false, false: true};
+		if(this.name){
+			update_paid_amount = update_paid_amount_dict[update_paid_amount]
```

Comment:
It can be written like this:

update_paid_amount = update_paid_amount ? false : true;